src/components/messenger/list/user-search-results/index.test.tsx

```diff
@@ -18,35 +23,54 @@ describe('UserSearchResults', () => {
   it('renders the title', function () {
     const wrapper = subject({});
 
-    expect(wrapper.find('.user-search-results__title').text()).toEqual('Start a new conversation:');
+    expect(wrapper.find(c('title')).text()).toEqual('Start a new conversation:');
```

Comment:
🌟

Reply:
bonus would be `toHaveText` :)

Reply:
> bonus would be `toHaveText` :)

https://github.com/zer0-os/zOS/pull/1478/commits/13934accd2b52b1aa819146d266df10f118e234b

😉